core/src/test/scala/SSLSpec.scala

```diff
@@ -0,0 +1,278 @@
+//: ----------------------------------------------------------------------------
+//: Copyright (C) 2015 Verizon.  All Rights Reserved.
+//:
+//:   Licensed under the Apache License, Version 2.0 (the "License");
+//:   you may not use this file except in compliance with the License.
+//:   You may obtain a copy of the License at
+//:
+//:       http://www.apache.org/licenses/LICENSE-2.0
+//:
+//:   Unless required by applicable law or agreed to in writing, software
+//:   distributed under the License is distributed on an "AS IS" BASIS,
+//:   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//:   See the License for the specific language governing permissions and
+//:   limitations under the License.
+//:
+//: ----------------------------------------------------------------------------
+
+package remotely
+
+import org.scalatest.matchers.{Matcher,MatchResult}
+import org.scalatest.{FlatSpec,Matchers,BeforeAndAfterAll}
+import java.io.File
+import Response.Context
+import transport.netty._
+import codecs._
+import scalaz.{\/-}
+import scalaz.stream.Process._
+
+class SSLSpec extends FlatSpec
+    with Matchers
+    with BeforeAndAfterAll {
+
+
+  behavior of "Netty SSL Server"
+  
+  val pems = List("CA.pem", "client_cert.pem", "server_cert.pem")
+  val keys = List("client_key.pk8", "server_key.pk8")
+
+  val caCert = new File(getClass.getResource("/ssl-testing/CA.pem").getFile)
+
+  val clientCert = new File(getClass.getResource("/ssl-testing/client_cert.pem").getFile)
+  val serverCert = new File(getClass.getResource("/ssl-testing/server_cert.pem").getFile)
+
+  val clientKey = new File(getClass.getResource("/ssl-testing/client_key.pk8").getFile)
+  val serverKey = new File(getClass.getResource("/ssl-testing/server_key.pk8").getFile)
+
+  val serverRequiringAuthParameters = SslParameters(Some(caCert),
+                                       Some(serverCert),
+                                       Some(serverKey),
+                                       None,
+                                       None,
+                                       None,
+                                       true)
+  val serverNoAuthParameters = SslParameters(Some(caCert),
```

Comment:
the fact that your asking this question is pointing more to a lack of documentation, but so this NoAuth version here will not require a client cert because the last boolean is false. 
The fact that we are passing Some(serverCert) means that it is going to be an SSL connection using this cert.

The Some(caCert) here will be unused, the fact that it is there means that we are going to setup a trustManager using this ca in order to valididate client certs, but since we aren't asking for any (because of the false) it will never be used.